%YAML:1.2

map:
  dim:                        [20.0, 20.0, 20.0]
  res:                        0.2

data:
  # # tsdf
  # truncation_boundary_factor: 8
  # max_weight:                 100

  # occupancy
  surface_boundary:           0
  min_occupancy:              -100
  max_occupancy:               100
  log_odd_min:                -5.015
  log_odd_max:                 5.015
  fs_integr_scale:            1
  const_surface_thickness:    false
  uncertainty_model:          linear
  tau_min_factor:             3
  tau_max_factor:             12
  k_tau:                      0.05
  sigma_min_factor:           1         # minimum/maximum uncertainty := 3 x sigma_min_factor/sigma_max_factor * voxel_dim
  sigma_max_factor:           3         # minimum/maximum uncertainty := 3 x sigma_min_factor/sigma_max_factor * voxel_dim
  k_sigma:                    0.05

sensor:
  width:                      720
  height:                     480
  fx:                         415.692194
  fy:                         415.692194
  cx:                         360.000000
  cy:                         240.000000
  left_hand_frame:            false
  near_plane:                 0.4
  far_plane:                  4.0

planner:

  # apartment scene
  min_x:                      -1.0
  max_x:                      25.0
  min_y:                      -5.0
  max_y:                      4.0
  min_z:                      -4.0
  max_z:                      4.0
  mav_radius:                 0.3
